iset.mm

```diff
@@ -59437,6 +59519,119 @@ the cancellation property (fifth hypothesis), show that the relation
+
+  ${
+    $d w x y z A $.  $d x B $.
+    $( The cardinality of a well-orderable set is an ordinal.  (Contributed by
+       Jim Kingdon, 30-Aug-2021.) $)
+    cardcl $p |- ( E. y e. On y ~~ A -> ( card ` A ) e. On ) $=
+      ( vx cen wbr con0 wrex ccrd cfv crab cint cvv cmpt wceq df-card a1i breq2
+      cv rabbidv wcel inteqd adantl simprd rexlimivw fvmptd onintrab2im eqeltrd
+      encv intexrabim ) ARZBDEZAFGZBHIUKAFJZKZFULCBUJCRZDEZAFJZKZUNLHLHCLURMNUL
+      CAOPUOBNZURUNNULUSUQUMUSUPUKAFUOBUJDQSUAUBUKBLTZAFUKUJLTUTUJBUHUCUDUKAFUI
+      UEUKAUFUG $.
+  $}
+
+  ${
+    $d A x y $.  $d B x y $.
+    $( A set equinumerous to an ordinal is numerable.  (Contributed by Mario
+       Carneiro, 29-Apr-2015.) $)
+    isnumi $p |- ( ( A e. On /\ A ~~ B ) -> B e. dom card ) $=
+      ( vy vx con0 wcel cen wbr wa crab cint cvv ccrd cdm wrex breq1 intexrabim
+      cv rspcev syl encv simprd wceq rabbidv inteqd eleq1d elrab3 adantl mpbird
+      wb breq2 df-card dmmpt syl6eleqr ) AEFZABGHZIZBCRZDRZGHZCEJZKZLFZDLJZMNUQ
+      BVDFZURBGHZCEJZKZLFZUQVFCEOVIVFUPCAEURABGPSVFCEQTUPVEVIUJZUOUPBLFZVJUPALF
+      VKABUAUBVCVIDBLUSBUCZVBVHLVLVAVGVLUTVFCEUSBURGUKUDUEUFUGTUHUIDLVBMDCULUMU
+      N $.
+  $}
+
+  ${
+    $d x y z A $.  $d x y B $.
+    $( Every finite set is numerable.  (Contributed by Mario Carneiro,
+       4-Feb-2013.)  (Revised by Mario Carneiro, 29-Apr-2015.) $)
+    finnum $p |- ( A e. Fin -> A e. dom card ) $=
+      ( vx cfn wcel cv cen wbr com wrex ccrd isfi con0 nnon ensym isnumi syl2an
+      cdm rexlimiva sylbi ) ACDABEZFGZBHIAJQDZBAKUAUBBHTHDTLDTAFGUBUATMATNTAOPR
+      S $.
+
+    $( Every ordinal number is numerable.  (Contributed by Mario Carneiro,
+       29-Apr-2015.) $)
+    onenon $p |- ( A e. On -> A e. dom card ) $=
+      ( con0 wcel cen wbr ccrd cdm enrefg isnumi mpdan ) ABCAADEAFGCABHAAIJ $.
+  $}
+
+  ${
+    $d A x y $.  $d A y z $.
+    $( The value of ` ( card `` A ) ` .  (Contributed by Jim Kingdon,
+       30-Aug-2021.) $)
+    cardval3ex $p |- ( E. x e. On x ~~ A ->
+        ( card ` A ) = |^| { y e. On | y ~~ A } ) $=
+      ( vz cv cen wbr con0 wrex cvv wcel crab cint ccrd cfv wceq encv rexlimivw
+      simprd breq1 cbvrexv sylbir breq2 rabbidv inteqd df-card fvmptg syl2anc
+      intexrabim ) AEZCFGZAHIZCJKZBEZCFGZBHLZMZJKZCNOUQPUKUMAHUKUJJKUMUJCQSRULU
+      OBHIURUOUKBAHUNUJCFTUAUOBHUIUBDCUNDEZFGZBHLZMUQJJNUSCPZVAUPVBUTUOBHUSCUNF
+      UCUDUEDBUFUGUH $.
+  $}
+
+  ${
+    $d x y z A $.
+    $( The value of the cardinal number function with an ordinal number as its
+       argument.  (Contributed by NM, 24-Nov-2003.)  (Revised by Mario
+       Carneiro, 13-Sep-2013.) $)
+    oncardval $p |- ( A e. On -> ( card ` A ) = |^| { x e. On | x ~~ A } ) $=
+      ( vy con0 wcel cv cen wbr wrex ccrd cfv crab cint wceq enrefg breq1 mpdan
+      rspcev cardval3ex syl ) BDEZCFZBGHZCDIZBJKAFBGHADLMNUABBGHZUDBDOUCUECBDUB
+      BBGPRQCABST $.
+
+    $( The cardinal of an ordinal number is less than or equal to the ordinal
```

Comment:
The cardinal -> The cardinality ?

Reply:
This language is in set.mm and there is no reason to diverge.

I suspect "the cardinal" more emphasizes the specific set which represents the result of ``card` and "cardinality" more represents the concept. But I'd be glad to defer to someone else if they want to come up with some kind of ruling on this terminology.

Reply:
OK, no change necessary.